backend/scripts/mongo/seedQuestions.js

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const { MongoClient, ObjectId } = require('mongodb');
+const questionsJson = require('../../data/questions.json');
+const fs = require('fs').promises;
+
+const generateFilePaths = async () => {
+  const questionDescFilePaths = [];
+
+  for (let i = 1; i <= questionsJson.length; i++) {
+    questionDescFilePaths.push(`data/question-descriptions/${i}.md`);
+  }
```

Comment:
Could we use the ID in the json file itself instead of the for loop variable? Just incase question ids and filenames are updated wrongly. Might need to update the seeding function too